### DISCUSSION TITLE
How to delete Supabase account?

### DISCUSSION THREAD

**Discussion by KariNarhi** [supabase/supabase#18513] [Questions] [answered]
The title says it all. 

Currently I cannot find any "Delete account" button.

Normally this should be under the Account/Preferences-page.

I just created the account for personal tinkering and learning about Supabase but for future reference I wanna know how to delete it.

Do I need to contact support this?
If so, then this should be a feature on the Account/Preferences so that users can delete their accounts themselves.

**Comment by GaryAustin1** [comment]
Contacting support is the only way currently.

**Reply by KariNarhi** [reply]
OK.

As I said, would be nice feature for users to be able to do it themselves.

I understand that this raises the issue with organizations/projects if other people or businesses are involved in those.

ps. Personally I think that this "Contact support" -method eats away some trust from users.
My inner alarms start ringing everytime I see that I cannot undo something myself when it is already done, like in this case after creating account and logging in, only to see that no Delete Account -button was present.

**Reply by GaryAustin1** [reply]
I agree.  Seems odd for them not to have this, I've not seen any reasoning either way.  You probably should write that in your support request.

**Comment by GaryAustin1** [comment, marked as answer]
This feature has now been added.
![image](https://github.com/supabase/supabase/assets/54564956/346b23fe-34af-4d77-b743-8ba3687b7a35)
